.eslintrc.json

```diff
@@ -11,6 +11,40 @@
   "plugins": ["prettier"],
   "rules": {
     "prettier/prettier": "error",
+
```

Comment:
why this blank line?

Reply:
Likely to separate style-only rules from syntax/logic error rules.

Reply:
can you add a comment? Depending on the JSON parser this should work

Reply:
Also the lonely `prefer-const` is not actually part of `eslint:recommended` at all, so it will never be folded into the other rules.

Reply:
You're right, ESLint allows comments in the JSON. So, added.